flask_monitoringdashboard/database/__init__.py

```diff
@@ -239,12 +239,30 @@ class CustomGraphData(Base):
     """Actual value that is measured."""
 
 class FullStackTrace(Base):
+    """Table for storing a hash of a stack trace, to avoid 'duplicate' ExceptionStackLines."""
+    
     __tablename__ = '{}FullStackTrace'.format(config.table_prefix)
     
     id = Column(Integer, primary_key=True)
     stack_trace_hash = Column(String(64), nullable=False, unique=True)
     
     stack_lines = relationship('ExceptionStackLine', backref='exception_info')
+
+class ExceptionType(Base):
+    """Table for storing Exception types"""
+    
+    __tablename__ = '{}ExceptionType'.format(config.table_prefix)
+    
+    id = Column(Integer, primary_key=True)
+    type = Column(String(1500), nullable=False)
```

Comment:
are we too generous with the length of this column? I think we are safe to keep it to 256 characters. if somebody uses exceptions longer than that, they can use a different exception monitor I'd say!

Reply:
LeChat agrees: 
<img width="954" alt="image" src="https://github.com/user-attachments/assets/0970e9b2-6ee6-411f-87ee-b691ea3ea7f4" />